src/styles/title-with-crown.scss

```diff
@@ -0,0 +1,23 @@
+.title-with-crown {
```

Comment:
what if title will be without crown?

Reply:
If it will without crown it will have not class title-with-crown

Reply:
it's too specific name, code should be reusable and flexible no matter what's on the image

Reply:
no. It is perfect name of titles with crown. If I will need other kind of titles I will make new class. But now there are not titles without crown

Reply:
imagine the crown is changed to some other type of element or completely removed
in this case it shouldn't be a problem and you don't have to change the document names just because you gave them too specific names, names should be flexible no matter what kind of pictures are inside

Reply:
I`m still disagree, but ok. I changed it